### ISSUE TITLE
[$500] Desktop/Web - IOU - CMD+Enter allows you to select users and WS into one Split Bill

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#29628]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.84-0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

## Action Performed:
Prerequisites:
Use an HT account that already has WS in it or create one in advance.
1. Open New Expensify app
2. Log into your HT account
3. Click on the FAB button
4. Click on 'Request Money'
5. Choose 'Manual' and add an amount
6. Click on the "Split" button next to WS
7. Click on the "Split" button next to any user
8. Press CMD+Enter(CTRL+Enter)

## Expected Result:
CMD+Enter commands should not work in forbidden cases such as Split Bill with normal user and WorkSpace at the same time

## Actual Result:
The CMD+Enter command takes you to the final Split Bill page, which includes normal users and WS at the same time, although this is not allowed by the system

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

 

https://github.com/Expensify/App/assets/78819774/0474a0c3-dcbf-49ee-ba98-b24cdb842efd

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @garrettmknight</details>

**Comment by garrettmknight** [comment]
Reproduced, opening up.

**Comment by situchan** [comment]
This bug will be gone after #27508

**Comment by garrettmknight** [comment]
Will keep an eye on https://github.com/Expensify/App/issues/27508

**Comment by garrettmknight** [comment]
Still holding on https://github.com/Expensify/App/issues/27508

**Comment by garrettmknight** [comment]
Still holding on https://github.com/Expensify/App/issues/27508

**Comment by garrettmknight** [comment]
Excellent - closing!

**Comment by lanitochka17** [comment]
Issue reproducible on the latest build 1.4.18-3

https://github.com/Expensify/App/assets/78819774/48343c50-d1b0-4c69-b5f9-e42704eefce1

**Comment by namhihi237** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Disable CTRL + ENTER when split bill disable button
### What is the root cause of that problem?
Currently, we do not have the logic disable CTRL ENTER same with ENTER here when the button passes `shouldDisableEnterShortcut` true
https://github.com/Expensify/App/blob/4ac7b3f0a529a13eeee0df4ea568777e5d4b2482/src/components/Button/index.tsx#L180-L185
### What changes do you think we should make in order to solve the problem?
We should implement like [this](https://github.com/Expensify/App/blob/4ac7b3f0a529a13eeee0df4ea568777e5d4b2482/src/components/Button/index.tsx#L180C1-L186C1) with the `CONST.KEYBOARD_SHORTCUTS.CTRL_ENTER`
Or If we want to separate 2 logic conditions enter and ctrl + enter we should add new param like `shouldDisableCtrlEnterShortcut`
### What alternative solutions did you explore? (Optional)
N/A

**Comment by pecanoro** [comment]
@hoangzinh Any accepted proposals that I need to review? I am a bit confused about what next steps are for this issue

**Comment by pecanoro** [comment]
Thanks, everyone! Assigning @neonbhai!

**Comment by garrettmknight** [comment]
@neonbhai all you!

**Comment by pecanoro** [comment]
@neonbhai Friendly reminder to create that PR since it's been a week already 😄

**Comment by pecanoro** [comment]
@garrettmknight It seems the issue was fixed by another PR when we were about to merge this PR and if I remember correctly, we still should issue payment since this wasn't the contributor's fault. Could you handle it?

**Comment by garrettmknight** [comment]
We've issued payment for work that turned out to be unnecessary, historically. I can do that this time, but @neonbhai I'd appreciate a little more urgency getting PRs up in the future. 🙇 Payment offer sent.

**Comment by garrettmknight** [comment]
Paid, closing.

**Comment by garrettmknight** [comment]
Sorry about that @hoangzinh - offer sent!
